tests/test_ocs_basic_install.py

```diff
@@ -181,3 +275,84 @@ def load_config_data(data_path):
     """
     with open(data_path, "r") as data_descriptor:
         return yaml.load(data_descriptor, Loader=yaml.FullLoader)
+
+
+def create_rook_resource(template_name, rook_data, cluster_path):
+    """
+    Create a rook resource after rendering the specified template with
+    the rook data from cluster_conf.
+
+    Args:
+        template_name: name of the ocs-deployment config template.
+        rook_data: rook specific config from cluster_conf
+        cluster_path: path to cluster directory, where files will be written
+    """
+    base_name = template_name.split('.')[0]
+    template_path = os.path.join('ocs-deployment', template_name)
+    template = render_template(
+        template_path,
+        rook_data.get(base_name, {})
+    )
+    cfg_file = os.path.join(cluster_path, template_name)
+    with open(cfg_file, "w") as f:
+        f.write(template)
+    log.info(f"Creating rook resource from {template_name}")
+    run_cmd(f"oc create -f {cfg_file}")
+
+
+def create_eb2_volumes(worker_pattern, size=100):
```

Comment:
rename to create_ebs_volume, I was probably confused with ec2 when I named it eb2. It's actually [EBS]( https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/EBSVolumes.html)

Reply:
fixed in https://github.com/red-hat-storage/ocs-ci/pull/23/commits/6659aa4f0c41c1972ffa9f7306ae8b7a8f4f52c1